### ISSUE TITLE
No License?

### ISSUE THREAD

**Issue by SnakeDoc** [jkransen/framboos#2]
Hello!

Excellent work on framboos! I'd like to use it in a fun side project of mine, however I can't seem to locate a license for the software in either the source files (boiler plate header) or a LICENSE file. 

Without an explicit license, it means the software automatically falls under the most restrictive form possible with all rights reserved, meaning I can't legally use it even for a fun side project. 

Will you consider licensing your code under an open source license of some sort? Perhaps Apache 2.0, MIT, or BSD licenses, etc ?

Thank you for your time,

Jason

**Comment by jkransen** [comment]
Hi Jason, absolutely, I always meant it to be open source. I'll pick a
licence right away, probably Apache 2.0. Consider it done, and use it for
anything you like. Could you tell me what you'll use it for?

Best wishes, Jeroen

2013/6/7 Jason Sipula notifications@github.com

**Comment by SnakeDoc** [comment]
Hello Jeroen,

Thank you very much for the response! I had imagined it was intended to be open source :-)

While I've been developing for a while, and been a systems admin for even longer, I'm just starting out with physical computing... so I'm new to using GPIO and the like. Your library is appealing because there is no other setup required as you use /sys/class/GPIO that comes default on most linux distros (especially on the RPi). Also your library is pretty simple, and I like simple! :+1: 

I plan to learn GPIO on the RPi by building a device that lights up some LED's depending on a detected temperature range. This is helpful because I'm lazy and don't like to get off the couch to check the thermostat! So a visual queue as to the room temperature range would be cool.

After I've learned the basics of GPIO on the RPi, the sky's the limit! My ultimate goal is to build a carputer from the RPi that can read the OBDII port and provide other "heads up" display information such as vehicle speed (by GPS) , etc.

I have also forked your repo so that I can take a deeper look at the code and possibly contribute back to the project.

Anyways, thanks for the response and I look forward to working with your library!
